include/main_functions_interactive.h

```diff
@@ -16,25 +16,26 @@
  * misspelled words, asking for correction choices from user through the command line and
  * replacing misspelled word with user choices
  * parameters:
- *      - line - line to be edited
- *      - dict - dictionary
- *      - linenumber - number of line
+ *      - line: line to be edited
+ *      - dict: dictionary
+ *      - linenumber: number of line
+ *		- returnflag: indicates if line being parsed is the last line (true if last line)
+ *			in this case, there is a formatting issue in printing out the line that needs to be resolved,
+ *			because the text file that was read does not have a newline character
+ *			at the end of the line.
  * return: edited line
  */
-char* edit_interactive(char* line, dict_t* dict, int linenumber, int returnflag);
+char *edit_interactive(char *line, dict_t *dict, int linenumber, bool returnflag, bool* color);
 
 /*
  * interactive_mode: open file to parse and store in list, edit each line, then return list
  * parameters:
  *      - filename: name of file to be parsed and corrected
  *      - dictionary: dictionary for searching 
  *      - quit: pointer to boolean (TRUE means continue in main.c loop, FALSE means exit program)
- *		- returnflag - indicates if line being parsed is the last line.
- *			in this case, there is a formatting issue in printing out the line that needs to be resolved,
- *			because the text file that was read does not have a newline character
- *			at the end of the file.
+ *      - color: pointer to boolean (TRUE means print in colors, FALSE means print without colors)
  * return: char** array of lines, to be printed in save page
  */
-char** interactive_mode(char* filename, dict_t* dict, bool* quit);
+char **interactive_mode(char *filename, dict_t *dict, bool *quit, bool* color);
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed!